support/build/config-submenu.sh

```diff
@@ -92,6 +104,39 @@ else
 	exec 7>&1
 fi
 
+#
+# GENERATE LIST OF FINAL LIBRARY PATHS
+#
+LIBS=()
+for ARG_PATH in "${ARG_PATHS[@]}" "$@"; do
+	# Remove trailing slashes
+	ARG_PATH="${ARG_PATH%/}"
+
+	# Does Config.uk exist?
+	CONFIG_UK="${ARG_PATH}/Config.uk"
+	if [ ! -f "${CONFIG_UK}" ]; then
+		if [ $OPT_QUIET -ne 0 ]; then
+			echo "Could not find \"Config.uk\" under \"${ARG_PATH}\". Skipping..." 1>&2
+		fi
+		continue
+	fi
+
+	DONT_INCLUDE=1
+	for ARG_EXCLSUFFIX in "${ARG_EXCLSUFFIXES[@]}"; do
+		if [[ "${ARG_PATH}" == *"${ARG_EXCLSUFFIX}" ]]; then
```

Comment:
Most important is that the matching result is identical with what happens in [`Makefile.rules`](https://github.com/unikraft/unikraft/pull/1104/files#diff-f535de85a92c17723a9b46772fd6d8d8252feb9802a5079056b253d1bdd01df3R246). This is the only way to keep `menuconfig` in sync with what the build system includes. This is why I kept the matching itself simple... I just noticed that I need to exclude also library-provided `Makefile.rules`... I'll add this to the PR update.

However, you are right that if you handover a trailing slash, this wont match at all anymore. I could add a comment about this in the help screen... what do you think?

Reply:
I think any tool must be self-contained and function correctly on any input. Will something break if we trim trailing slashes similarly to what it does on line 113?

Reply:
It won't break, we just would need to keep the matching operation in sync within the [main makefile](https://github.com/unikraft/unikraft/pull/1104/files#diff-f535de85a92c17723a9b46772fd6d8d8252feb9802a5079056b253d1bdd01df3R246). What I mean is that we need to remove the trailing slash there, too. I'd need to check how this can be achieved. On the other hand, the exclude suffix is handed over by the user and then used by the build system and forwarded to `config-submenu.sh` calls. Because of this I meant the behavior must be in sync at least. At the moment both would not match with anything anymore if a trailing slash was handed over.

Reply:
I updated the PR to remove a trailing slash from paths